src/platform_impl/linux/common/keymap.rs

```diff
@@ -0,0 +1,882 @@
+//! Convert Wayland keys to winit keys.
```

Comment:
Not sure why `Wayland` is here given that it's a common thing?

Reply:
I'm not sure why that's like that either. "X11-style" is what I'd go with (since it existed first), but as you mention, it's the same both on X11 and Wayland.